csv-validator-application/assembly.xml

```diff
@@ -0,0 +1,89 @@
+<!--
```

Comment:
Should this be renamed to `assembly-nix-with-jre.xml` to be consistent with the nameing of the Windows specific file?

Reply:
replied in a thread elsewhere